internal/store/static.go

```diff
@@ -78,7 +78,7 @@ func (store *StaticStore) GetPlugins(pluginFilter *model.PluginFilter) ([]*model
 		return nil, nil
 	}
 
-	plugins, err := store.getPlugins(pluginFilter.ServerVersion)
+	plugins, err := store.getPlugins(pluginFilter.ServerVersion, pluginFilter.EnterprisePlugins)
```

Comment:
It's still weird to me that the filter parameter is `EnterprisePlugins` but then we go ahead and call it `includeEnterprisePlugins` in the receiving function. Worth converging on the same terminology across the board?

Reply:
My reasoning behind the naming was that from the perspective of the server `EnterprisePlugins` only indicates that the server has the license to run Enterprise Plugins. He decision to also deliver these plugins is done my the Marketplace and hence the name here is `includeEnterprisePlugins`. In the future, the Marketplace might react differently.